### PR TITLE
Code fixes and formatting

### PR DESCRIPTION
**Pull request by EmeraldBlock**
Fixes and formatting for the code examples and the code snippets in the curriculum.

### CODE REVIEW

File: CS_Ed_Week/code-examples/repeat_string.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
-def Repeat_String(input_number, input_string):
+def repeat_string(input_number, input_string):
   temp_list = []
-  for x in range(input_number):
+  for _ in range(input_number):
```
Comment (cuppajoe123) [inline CS_Ed_Week/code-examples/repeat_string.py:3]: I think having an underscore as the variable name is sort of confusing. I think we should stick with alphabet characters or mixtures of alphabet, numbers, and symbols for variables names, and not single symbols, as that could be mistaken for an operator of some sort.

Comment (cuppajoe123) [inline CS_Ed_Week/code-examples/repeat_string.py:3]: I think having an underscore as the variable name is sort of confusing. I think we should stick with alphabet characters or mixtures of alphabet, numbers, and symbols for variables names, and not single symbols, as that could be mistaken for an operator of some sort.

**Review by cuppajoe123 — COMMENTED** [review]
Looks good, but the underscore as a variable name in repeat_string.py seems inconsistent with the rest of the variable names.

File: CS_Ed_Week/code-examples/repeat_string.py (hunk 1 above)
Comment (EmeraldBlock) [reply CS_Ed_Week/code-examples/repeat_string.py:3]: this is common practice for unused "throwaway" variables, I don't think it's that hard to explain it's just a variable but I guess it's also not too necessary

**Pull request merged**